projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,7 +18,49 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const getConnectionsDiv = () => {
+	// eslint-disable-next-line no-undef
+	if ( ! connections ) {
+		return null;
+	}
+	// eslint-disable-next-line no-undef
+	const providers = Object.keys( connections );
+	return providers.map( provider => {
+		// eslint-disable-next-line no-undef
+		const connectionIds = Object.keys( connections[ provider ] );
+		const connectionDiv = connectionIds.map( connectionId => {
+			// eslint-disable-next-line no-undef
+			const connection = connections[ provider ][ connectionId ];
+			return (
```

Comment:
Here, I'd also recommend defining a separate component higher up in the file, something like:

```js
const ConnectionItem = ({ connection }) => (
	<div>...all connection item html here...</div>
)
```

Then, in your loop you can do this: `<ConnectionItem connection={ connections[ provider ][ connectionId ] } />`